Wox.Plugin/SharedCommands/FilesFolders.cs

```diff
@@ -97,5 +97,15 @@ public static void RemoveFolder(this string path)
 #endif
             }
         }
+
+        public static bool LocationExists(this string path)
```

Comment:
this is never used, why do we need this?

Reply:
This is used:
![image](https://user-images.githubusercontent.com/26427004/77245810-91b8c400-6c75-11ea-8e2f-8ccbc690cdb3.png)